### DISCUSSION TITLE
Whether the LightGBM model is pre-downloaded into the pytorch environment

### DISCUSSION THREAD

**Discussion by perveil** [PaddlePaddle/PaddleSpatial#161] [Q&A] [answered]
in my method ，i will use LightGBM and pytorch simultaneously，when i submit ，i find the some error which mean can not load  the prepare.py

**Comment by seeingisbelieveing** [comment]
Dear perveil, could you please provide your team name and submission date? We will check it, Thx.

**Comment by shaido987** [comment, marked as answer]
I've used LightGBM successfully within the pytorch environment (it is also listed in the updated environment packages [here](https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/pytorch_env_installed_packages.md)). You should have some other issue within the code.
